content/documentation/edge/installation-guide/index.adoc

```diff
@@ -153,8 +153,30 @@ It is not usually necessary to explicitly execute the link:https://github.com/ki
   bash <(curl -L https://kiali.io/getLatestKialiOperator) --help
 ----
 
-icon:bullhorn[size=1x]{nbsp} The install script requires `envsubst` installed and in your PATH; you can get it via the GNU `gettext` package.
+==== OperatorHub
+
+For production environments that have OperatorHub installed (such as OpenShift), you will want to install Kiali Operator using OperatorHub. Simply go to the OperatorHub console and install Kiali Operator. At that point, you can link:#_create_or_edit_the_kiali_cr[create the Kiali CR] to install Kiali.
+
+==== Helm Chart
+
+For production environments that do not have OperatorHub, you will most likely want to use link:https://kiali.org/kiali-operator/charts/index.yaml[the Helm Chart] rather than the install script. In fact, the install script uses the Helm Chart to perform the installation.
```

Comment:
maybe the link text should be [the Kiali Operator Helm Chart]

Reply:
will be done